### PR TITLE
Update module.md with queryRow() documentation

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose

$subject

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/1914

## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: ballerina/Module.md [hunk 1]
```diff
@@ -213,24 +213,21 @@ error? e = resultStream.forEach(function(record{} student) {
...
-} else {
-    // Student table must be empty.
-}
+int id = 10;
+sql:ParameterizedQuery query = `SELECT * FROM students WHERE id = ${id}`;
+Student retrievedStudent = check dbClient->queryRow(query);
+```
 
-error? e = resultStream.close();
+The `queryRow()` operation can also be used to retrieve a single value from the database (e.g. when querying using
```
Comment (praneesha) [inline ballerina/Module.md:224]:
```suggestion
The `queryRow()` operation can also be used to retrieve a single value from the database (e.g., when querying using 
```

File: ballerina/Module.md [hunk 2]
```diff
@@ -213,24 +213,21 @@ error? e = resultStream.forEach(function(record{} student) {
...
-    // Student table must be empty.
-}
+int id = 10;
+sql:ParameterizedQuery query = `SELECT * FROM students WHERE id = ${id}`;
+Student retrievedStudent = check dbClient->queryRow(query);
+```
 
-error? e = resultStream.close();
+The `queryRow()` operation can also be used to retrieve a single value from the database (e.g. when querying using 
+`COUNT()` and other SQL aggregation functions). If the provided return type is not a record (i.e. a primitive data type)
```
Comment (praneesha) [inline ballerina/Module.md:225]:
```suggestion
`COUNT()` and other SQL aggregation functions). If the provided return type is not a record (i.e., a primitive data type)
```

Comment (kaneeldias) [reply ballerina/Module.md]: Fixed in bc4f616

File: ballerina/Module.md (hunk 1 above)
Comment (kaneeldias) [reply ballerina/Module.md]: Fixed in bc4f616

**Review by daneshk — APPROVED** [review]

**Review by niveathika — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**